client/src/pages/recording-view/recording-view.tsx

```diff
@@ -60,6 +48,38 @@ export function RecordingViewPage() {
       setIQData(displayedIQ);
     }
   }, [displayedIQ]);
+  return (
+    <>
+      <Stage width={spectrogramWidth + 110} height={30}>
```

Comment:
Would these static numbers cause problems with different screen sizes?

Reply:
I know thye have been default in the previous component as logn as we keep the size of the rulers sttic theu are ok but yeah it is something we will need ot review later.